src/api/restApi.ts

```diff
@@ -43,6 +43,7 @@ export class RestApi {
   private readonly _post = (url: string, body: any) =>
     this.wretcher()
       .url(url)
+      .headers({SignatureVerificationToken: 'asdasdasd'})
```

Comment:
this injection should be separated from the general method, i.e. declared in another method, or injected as param like `additionalHeaders`